includes/admin/tools/data/class-give-tools-reset-stats.php

```diff
@@ -128,14 +128,14 @@ public function get_data() {
 						$sql[] = "DELETE FROM $wpdb->postmeta WHERE post_id IN ($ids)";
 						$sql[] = "DELETE FROM $wpdb->comments WHERE comment_post_ID IN ($ids)";
 						$sql[] = "DELETE FROM $wpdb->commentmeta WHERE comment_id NOT IN (SELECT comment_ID FROM $wpdb->comments)";
-						$sql[] = "DELETE
+						$sql[] = $wpdb->prepare( "DELETE
 							FROM $wpdb->terms
 							WHERE $wpdb->terms.term_id IN
 							(SELECT $wpdb->term_taxonomy.term_id
 							FROM $wpdb->term_taxonomy
-							WHERE $wpdb->term_taxonomy.taxonomy = 'give_forms_category'
-							OR $wpdb->term_taxonomy.taxonomy = 'give_forms_tag')";
-						$sql[] = "DELETE FROM $wpdb->term_taxonomy WHERE $wpdb->term_taxonomy.taxonomy = 'give_forms_category' OR $wpdb->term_taxonomy.taxonomy = 'give_forms_tag'";
+							WHERE $wpdb->term_taxonomy.taxonomy = '%s'
```

Comment:
@Sidsector9 please read doc how to use `wpdb`: https://codex.wordpress.org/Class_Reference/wpdb#Examples